llm_bench/python/utils/conversion_utils/helpers.py

```diff
@@ -157,7 +157,7 @@ def get_data_aware_args(ov_model, tokenizer, config, compression_args, args):
         dataset_args = compression_args['dataset']
         dataset_params = dataset_args['name']
         if 'sensitivity_metric' in dataset_args:
-            res['mode'] = dataset_args['sensitivity_metric']
+            res['sensitivity_metric'] = dataset_args['sensitivity_metric']
```

Comment:
@andreyanufr it shouldn't be `mode`, right?

Reply:
res['sensitivity_metric'] = dataset_args['sensitivity_metric'] is right option